modules/ROOT/pages/index.adoc

```diff
@@ -53,9 +53,9 @@ You can enable monitoring and alerting in AWS or Azure | Included
 
 == Anypoint Runtime Fabric and Standalone Mule Runtimes (Hybrid Deployments)
 
-Hybrid deployments of Mule applications require you to install a version of the Mule runtime on a server and deploy one or more applications on the server. Each application shares the Mule runtime and the resources made available to it. Other resources such as certificates or database connections may also be shared using domains.
+Hybrid deployments of Mule applications require you to install a version of the Mule runtime on a server and deploy one or more applications on the server. Each application shares the Mule runtime server and the resources allocated to it. Other resources such as certificates or database connections may also be shared using domains.
 
-Anypoint Runtime Fabric provisions resources differently. Each Mule application and API gateway runs within their own Mule runtime and in their own container. The resources the container can access is specified when deploying a Mule application or API proxy. This enables Mule applications to horizontally scale across VMs without relying on other dependencies. It also ensures that different applications do not compete with each other for resources on the same VM.
+Anypoint Runtime Fabric provisions resources differently. Each Mule application and API gateway runs within its own Mule runtime and in its own container. The resources available to the container are specified when deploying a Mule application or API proxy. This enables Mule applications to horizontally scale across VMs without relying on other dependencies. It also ensures that different applications do not compete with each other for resources on the same VM.
```

Comment:
Hey @sundippatel Can you review this paragraph? Looks like we might want to reword since it mentions VMs.

Reply:
I think we can sub "node" for "VM". 
For more clarity, perhaps we can, 
- on the first instance of "node", say something like "to horizontally scale across VMs, or nodes, without relying on other dependencies."
- then we could just use "node" for following sentences.

Reply:
Incorporated these comments after PR was merged.